build/config/android/abi.gni

```diff
@@ -34,13 +34,18 @@ declare_args() {
   # Forces all APKs/bundles to be 64-bit only to improve build speed in the CQ
   # (no need to also build 32-bit library).
   skip_secondary_abi_for_cq = false
+
```

Comment:
please undo this and check your submission carefully before pr

Reply:
> please undo this. I don't see any changes required for this.

Got it.